packages/radix-ui-themes/src/components/base-button.css

```diff
@@ -156,6 +156,13 @@
       background-image: linear-gradient(var(--black-a3), transparent, var(--white-a2));
     }
   }
+  /* Better -webkit-tap-highlight-color */
+  @media (hover: none) {
+    &:where(:active:not([data-state='open'])) {
+      outline: 0.5em solid var(--accent-a4);
+      outline-offset: 0;
```

Comment:
Not using `-webkit-tap-highlight-color` because it lags behind the touch interactions